daprdocs/content/en/reference/cli/dapr-mtls/dapr-mtls-renew-certificate.md

```diff
@@ -0,0 +1,73 @@
+---
+type: docs
+title: "mtls renew certificate CLI command reference"
+linkTitle: "mtls renew certificate"
+description: "Detailed information on the mtls renew certificate CLI command"
+weight: 3000
+---
+
+### Description
+This command can be used to renew expiring Dapr certificates in Kubernetes cluster.
+It renews root CA certificate, issuer certificate and issuer key.
+
+### Supported platforms
+
+- [Kubernetes]({{< ref kubernetes >}})
+
+### Usage
+
+```bash
+dapr mtls renew-certificate [flags]
+```
+
+### Flags
+
+| Name           | Environment Variable | Default           | Description                                 |
+| -------------- | -------------------- | ----------------- | ------------------------------------------- |
+| `--help`, `-h` |                      |                   | help for renew-certificate
+| `--kubernetes`, `-k` |                      | `false` | supprted platform|                             |
+| `--valid-until`  |                      | 365 days |  Validity for newly created certificates |
+| `--restart`  |                      | false |  Restarts Dapr control plane services (Sentry service, Operator service and Placement server) |
+| `--timeout`  |                      | 300 sec |  The timeout for the certificate renewal process |
```

Comment:
I do not understand this timeout. This is the time for the renewal process to complete? What if this is a 300 node K8s cluster, will this succeed?

Reply:
This is actually default Helm wait timeout for any process like install/upgrade to complete. If any operation is supposed to take more time, we can set this flag to higher value to stop timing out helm operation. see `helm install/upgrade -h`